.github/workflows/yehezgun.yml

```diff
@@ -0,0 +1,38 @@
+name: Template Git Actions
+
+run-name:
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Base-Actions:
```

Comment:
I think it's better to use lowercase letters because usually CI jobs uses them as a convention.

Reply:
Updated to `base-actions`